src/templates/advance-search-template.js

```diff
@@ -59,9 +59,9 @@ export default function searchByPropertiesModalTemplate() {
           }
         }"
       > 
-        <span class="upper bold-text method-fg ${path.method}">${path.method}</span> 
-        <span>${path.path}</span>
-        <span class="regular-font gray-text">${path.summary}</span>
+        <span data-content-id='${path.elementId}' class="upper bold-text method-fg ${path.method}">${path.method}</span> 
```

Comment:
instead of adding `data-content-id='${path.elementId}'` can you try by adding `style="pointer-events: none"`
`Id`s in general shouldn't be duplicated.

Reply:
Thank you @mrin9 , makes sense , that worked as well , will push it soon.

Thanks again.